lib/Dialect/TritonGPU/Transforms/Pipeliner/MatmulLoopPipeline.cpp

```diff
@@ -660,6 +644,87 @@ bool mlir::triton::preProcessLoopAndGetSchedule(
   return true;
 }
 
+/// Find the minimum number of insert_slice ops between the extract
+/// and the insert. This is equivalent to counting async_commit_group ops
```

Comment:
Correct, these are equivalent today because we always `async_commit_group` for every `insert_slice`, but if in the future we do:
`insert_slice
insert_slice
commit_group`
then this assumption would be invalid. But I don't see why we would want to do that.

Reply:
Good point, I'll mention it in the comment!

Reply:
ah yeah we should be counting `async_commit_group`, it shouldn't be more difficult right? Let's just do it instead of relying on assumptions

Reply:
Agreed

Reply:
Done!